src/timeline.js

```diff
@@ -605,7 +615,7 @@ export default class Timeline extends React.Component {
       this._selectRectangleInteractable
         .draggable({
           enabled: true,
-          ignoreFrom: '.item_draggable, .rct9k-group',
+          ignoreFrom: '.item_draggable, .rct9k-group'
```

Comment:
I see format changes appeared again. I also understand why: their automatic checks don't have the same conventions as VS code. I suggest to apply formatting only locally (via selection).